nixos/modules/services/web-apps/nextcloud.xml

```diff
@@ -63,17 +65,22 @@
    are used internally to configure an HTTP server using
    <literal><link xlink:href="https://php-fpm.org/">PHP-FPM</link></literal>
    and <literal>nginx</literal>. The <literal>config</literal> attribute set is
-   used for the <literal>config.php</literal> which is used for the
-   application's configuration. <emphasis>Beware: this isn't entirely pure
-   since the config is modified by the application's runtime!</emphasis>
+   used by the imperative installer and all values are written to an additional file
```

Comment:
One can still install nextcloud imperatively?

Reply:
When deploying Nextcloud the first time, the config will be generated using their installer tool (which is an imperative tool). After that it's currently not possible to change e.g. the database configuration.